base/timing.jl

```diff
@@ -56,8 +56,16 @@ function gc_alloc_count(diff::GC_Diff)
 end
 
 # cumulative total time spent on compilation, in nanoseconds
-cumulative_compile_time_ns_before() = ccall(:jl_cumulative_compile_time_ns_before, UInt64, ())
-cumulative_compile_time_ns_after() = ccall(:jl_cumulative_compile_time_ns_after, UInt64, ())
+function cumulative_compile_time_ns_before()
+    comp = ccall(:jl_cumulative_compile_time_ns_before, UInt64, ())
+    recomp = ccall(:jl_cumulative_recompile_time_ns_before, UInt64, ())
+    return comp, recomp
+end
+function cumulative_compile_time_ns_after()
+    comp = ccall(:jl_cumulative_compile_time_ns_after, UInt64, ())
+    recomp = ccall(:jl_cumulative_recompile_time_ns_before, UInt64, ())
```

Comment:
Should this be `_after`?

Also, why do we need `_before` and `_after`? They seem to do the same thing.

Reply:
True. `_before` etc. was because the functions also enabled/disabled the timings. But given there's another number to handle now it makes sense instead to separate enabling/disabling timing and getting the numbers

Reply:
I just tidied up the functions in line with this